src/ui/page/error/renderingError/renderingError.scss

```diff
@@ -0,0 +1,131 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-rendering-error-page-main {
+  @include with-theme() {
+    height: 100%;
+
+    &.dark {
+      background: center / cover url('@/ui/asset/images/rendering-error-dark.webp');
+
+      @include use-breakpoints(('mobile')) {
+        background: unset;
+      }
+
+      @include use-breakpoints(('tablet')) {
+        background: center / contain url('@/ui/asset/images/rendering-error-dark.webp') no-repeat;
+      }
+
+      .okp4-dataverse-portal-rendering-error-page-text-wrapper {
+        @include use-breakpoints(('mobile')) {
+          background: center url('@/ui/asset/images/rendering-error-dark-mobile.webp') no-repeat;
+        }
+      }
+    }
+
+    &.light {
+      background: center / cover url('@/ui/asset/images/rendering-error-light.webp');
+
+      @include use-breakpoints(('mobile')) {
+        background: unset;
+      }
+
+      @include use-breakpoints(('tablet')) {
+        background: center / contain url('@/ui/asset/images/rendering-error-light.webp') no-repeat;
```

Comment:
I think you don't need this as it is the same background

Reply:
Just tested it and it works! Thanks!